.github/workflows/rust.yml

```diff
@@ -14,74 +14,33 @@ jobs:
     name: Check
     runs-on: ubuntu-24.04
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
```

Comment:
see https://github.com/dtolnay/rust-toolchain/issues/133#issuecomment-2610825530

explicit toolchain dependencies are automatically managed by `rustup` (available in all default gh runners) through `rust-toolchain.toml`